qiita_db/data.py

```diff
@@ -1173,3 +1332,39 @@ def processed_date(self):
         return conn_handler.execute_fetchone(
             "SELECT processed_date FROM qiita.{0} WHERE "
             "processed_data_id=%s".format(self._table), (self.id,))[0]
+
+    @property
+    def status(self):
+        conn_handler = SQLConnectionHandler()
+        sql = """SELECT pds.processed_data_status
+                FROM qiita.processed_data_status pds
+                  JOIN qiita.processed_data pd
+                    ON pd.processed_data_status_id=pds.processed_data_status_id
+                WHERE pd.processed_data_id=%s"""
+        return conn_handler.execute_fetchone(sql, (self._id,))[0]
+
+    @status.setter
+    def status(self, status):
```

Comment:
There is no sanity check for the status, so when I tested it with status 'chicken' it errors out. Better to use `convert_to_id` and let that error. This goes for all the other status setters.